tmeister/static/src/toggles/table/toggle-table.helpers.js

```diff
@@ -29,6 +29,7 @@ export function getEnvList(toggles) {
 
 export function groupTogglesByFeature(toggles, envList) {
   return chain(toggles)
+    .sortBy((t) => t.toggle.feature)
```

Comment:
There was an issue with the list resorting when toggles changed. This fixes that.